itou/www/job_seekers_views/forms.py

```diff
@@ -40,6 +61,69 @@ def __init__(self, job_seeker_qs, data, *args, request_user, **kwargs):
             if job_seeker.get_full_name()
         ]
 
+    def _get_choices_for_organization_members(self, job_seeker_qs, request_organization):
+        """
+        Get members of the current user's organization, present or past,
+        that created or applied for a job seeker.
+        """
+        created_by_id = [job_seeker.created_by_id for job_seeker in job_seeker_qs]
```

Comment:
```suggestion
        created_by_id = [job_seeker.created_by_id for job_seeker in job_seeker_qs if job_seeker.created_by_id]
```
et je pense que ça permet de se débarasser du `- set([None])` en dessous.

Reply:
Et au passage, vu qu'on veut un set, autant le créer directement: {job_seeker.created_by_id for job_seeker in job_seeker_qs if job_seeker.created_by_id}

Reply:
Je me demandais si utiliser `set` ne serait pas plus performant, mais la différence ne doit pas casser des briques.